### PR TITLE
network dropdown selector

### LINKED ISSUE #273: Make network a drop down selector

### PR DESCRIPTION
**Pull request by benalleng**
Should close #273 added all 4 networks available

### CODE REVIEW

File: frontend/src/components/SettingStringsEditor.tsx [hunk 1]
```diff
@@ -36,12 +37,27 @@ export default function SettingStringsEditor() {
...
             <p className="text-2xl font-light">Don't trust us! Use your own servers to back Mutiny</p>
             <div className="flex flex-col gap-2 w-full">
                 <h3 className="text-lg font-light uppercase mt-2">Network</h3>
-                <input onChange={handleInputChange("network")} defaultValue={nodeManagerSettings.network} className={`w-full ${inputStyle({ accent: "blue" })}`} type="text" placeholder='Network' />
+                <form>
+                    <select onChange={handleSelectChange("network")} className={`w-full text-black bg-white text-xl ${inputStyle({ accent: "blue" })}`} name='' value={selectedNetwork} placeholder="Network">
+                        <option className="text-base" value="bitcoin">bitcoin</option>
+                        <option className="text-base" value="testnet">testnet</option>
+                        <option className="text-base" value="regtest">regtest</option>
+                        <option className="text-base" value="signet">signet</option>
```
Comment (benthecarman) [inline frontend/src/components/SettingStringsEditor.tsx:58]: Can we make these displayed as 'Mainnet', 'Testnet', 'Signet', and 'Regtest', also in that order.

**Review by benthecarman — APPROVED** [review]
Lgtm!

**Comment by benalleng** [comment]
I noticed some odd behavior if you change the network and then cancel when the warning prompt is shown when saving

**Review by futurepaul — CHANGES_REQUESTED** [review]
nice quality of life change! please change the styling to match the amount currency selector:
(replace the existing style with `className={selectStyle({ accent })}`)

**Review by futurepaul — APPROVED** [review]

**Pull request merged**